master/changes.html

```diff
@@ -883,6 +883,9 @@ <h3 id="interact">Scripting and Interactivity chapter (separate chapters in SVG
   the 'cursor element' element to take an URL that is not in a CSS-like
   functional form.</li>
   <li>Added the <span class="prop-value">bounding-box</span> keyword to <a>'pointer-events'</a>.</li>
+  <div class='changed-since-cr1 cr2'>
```

Comment:
Shouldn't that be

```
<div class='changed-since-cr1 cr2 cr3'>
```

Reply:
We indeed have 3 CRs already. Hm. There might be issues in other, already merged PRs as well. Will take a look this week.